SmartDeviceLink/private/SDLMenuManager.m

```diff
@@ -645,11 +645,14 @@ - (SDLAddCommand *)sdl_commandForMenuCell:(SDLMenuCell *)cell withArtwork:(BOOL)
     params.menuName = cell.title;
```

Comment:
It looks like the secondary artworks are not being uploaded. In the `sdl_findAllArtworksToBeUploadedFromCells:` needs to check if the `cell.secondaryArtwork` needs to be uploaded and, if so, add it to the list if artworks that need to be uploaded. You should probably also check if the `ImageFieldName` of `menuCommandSecondaryImage` or `menuSubMenuSecondaryImage` was returned in the `windowCapability` before uploading the artwork to avoid unnecessary uploads. You should be able to use the existing `hasImageFieldOfName` on the `SDLWindowCapability+ScreenManagerExtensions.h`